src/sidebar/components/group-list-item-out-of-scope.js

```diff
@@ -1,37 +1,106 @@
 'use strict';
 
+const classnames = require('classnames');
+const { Fragment, createElement } = require('preact');
+const { useState } = require('preact/hooks');
+const propTypes = require('prop-types');
+
 const {
   orgName,
   trackViewGroupActivity,
 } = require('../util/group-list-item-common');
 
-// @ngInject
-function GroupListItemOutOfScopeController(analytics) {
-  // Track whether the group details are expanded.
-  this.isDetailsExpanded = false;
+function GroupListItemOutOfScope({ analytics, group }) {
+  const [isExpanded, setExpanded] = useState(false);
 
-  /**
-   * Toggle the expanded setting on un-selectable groups.
-   */
-  this.toggleGroupDetails = function(event) {
+  const toggleGroupDetails = event => {
     event.stopPropagation();
-    this.isDetailsExpanded = !this.isDetailsExpanded;
-  };
-
-  this.orgName = function() {
-    return orgName(this.group);
+    setExpanded(!isExpanded);
   };
 
-  this.trackViewGroupActivity = function() {
+  const groupOrgName = orgName(group);
+  const trackViewActivity = event => {
+    event.stopPropagation();
     trackViewGroupActivity(analytics);
   };
+
+  return (
+    <div
+      className="group-list-item__item group-list-item-out-of-scope__item"
+      onClick={toggleGroupDetails}
+      tabIndex="0"
+    >
+      {/* the group icon */}
+      <div className="group-list-item__icon-container">
+        {group.logo && (
+          <img
+            className="group-list-item__icon group-list-item__icon--organization"
+            alt={groupOrgName}
+            src={group.logo}
+          />
+        )}
+      </div>
+      {/* the group name */}
+      <div
+        className={classnames({
+          'group-list-item-out-of-scope__details': true,
+          expanded: isExpanded,
+        })}
+      >
+        <svg
```

Comment:
Might be nice to have some sort commenting somewhere, if possible, about what this SVG _is_

Reply:
Done. I've extracted this into a separate, named, constant.